### DISCUSSION TITLE
When connected to Wi-Fi then toogle off / on

### DISCUSSION THREAD

**Discussion by dusanhauko** [arendst/Tasmota#20193] [Q&A]
Hello,
I hope anyone can help me. I'm basicaly making my car port a little bit smarter about my cars.
I have Sonoff SV flashed to tasmota and mounted on car's battery.

I'd like to configure Tasmota on device as follows:
- when device connects to my wifi
- then trigger device output for sure to OFF and back to ON (nothing connected; just for monitoring purposes in Smartthings) 

I think [PowerOnState] is useless in this idea as the device is allways online.
Any help?

Dusan H.

**Comment by barbudor** [comment]
Read docs about rules using trigger 'on wifi#connected'
